src/components/presentational/ValueSelector/ValueSelector.css

```diff
@@ -0,0 +1,77 @@
+.mdhui-value-selector {
+    --mdhui-value-selector-title-color: var(--mdhui-text-color-1);
+    --mdhui-value-selector-subtitle-color: var(--mdhui-text-color-3);
+    --mdhui-value-selector-value-background-color: #fff;
+    --mdhui-value-selector-value-text-color: var(--mdhui-text-color-1);
+    --mdhui-value-selector-button-value-selected-background-color: var(--mdhui-color-primary);
+    --mdhui-value-selector-button-value-selected-text-color: #fff;
+    --mdhui-value-selector-checkbox-color: var(--mdhui-text-color-4);
+    --mdhui-value-selector-checkbox-selected-color: var(--mdhui-color-primary);
+    padding: 16px;
+}
+
+.mdhui-value-selector-title {
+    font-weight: bold;
+    font-size: 18px;
```

Comment:
Would always use ems ...and I would stick to sizes consistent with other components even if it is a little off from the design...so that we are consistent.  I think for instance our default text size (e.g. 1em) ends up being 17px not 18px but 12px is commonly used

Reply:
(using ems lets iOS dynamic text work across everything though, e.g. if someone bumps up their desired font size)